lib/new_relic/rack/browser_monitoring.rb

```diff
@@ -23,8 +23,8 @@ class BrowserMonitoring < AgentMiddleware
       CONTENT_TYPE = 'Content-Type'.freeze
       CONTENT_DISPOSITION = 'Content-Disposition'.freeze
       CONTENT_LENGTH = 'Content-Length'.freeze
-      ATTACHMENT = 'attachment'.freeze
-      TEXT_HTML = 'text/html'.freeze
+      ATTACHMENT = /attachment/.freeze
+      TEXT_HTML = %r{text/html}.freeze
```

Comment:
TIL about %r{}! Nice

Reply:
It's most useful for cases like this where there's a literal forward slash character involved in the pattern.